history/history-network.md

```diff
@@ -20,7 +20,7 @@ In addition, the chain history network provides individual epoch records for the
     - Ommers
     - Withdrawals
 - Receipts
-- Header epoch records (pre-merge only)
+- Block header indexes
```

Comment:
I would remove this as it's actually the block headers that are provided, but accessed by number